runtime/proto/service/log_replication.proto

```diff
@@ -52,4 +54,32 @@ enum LogReplicationEntryType {
   SNAPSHOT_REPLICATED = 4;
   SNAPSHOT_END = 5;
   SNAPSHOT_TRANSFER_COMPLETE = 6;
+}
+
+enum ReplicationModel {
+  FULL_TABLE = 0;                 // Full table replication (Single Source to Single Sink = 1:1)
+  ROUTING_QUEUES = 1;             // Routing queue replication (used for entry-level replication) (1:1)
+  LOGICAL_GROUPS = 2;             // Table association to logical group (1:1)
+  MULTI_SOURCE_MERGE = 3;         // Same table replication from multiple sources to single sink (n:1)
+}
+
+message ClientInfoKey {
+  ReplicationModel model = 1;
+  string client_name = 2;
+  string group_name = 3;
+}
+
+message SinksInfoVal {
+  option (org.corfudb.runtime.table_schema).stream_tag = "lr_sessions";
+  repeated string destination_ids = 1;
+}
+
+message LRClientId {
+  string client_name = 1;
+}
+
+message LRClientInfo {
+  string client_name = 1;
+  ReplicationModel model = 2;
+  google.protobuf.Timestamp registration_time = 3;
```

Comment:
How do we plan on using the time?

Reply:
I'm not sure what the intended use is, but I went ahead and included it based on Anny's suggestion.